Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -1458,7 +1458,15 @@ private extension ChatChannelController {
     }
 
     private func getFirstUnreadMessageId() -> MessageId? {
-        guard let lastReadMessageId = channel?.reads.first(where: { $0.user.id == client.currentUserId })?.lastReadMessageId else {
+        let messages = self.messages.filter {
+            $0.type == .regular || $0.type == .reply || $0.type == .deleted
+        }
```

Comment:
When getting the first unread message, we need to skip system, ephemeral and error messages